tests/console/snapper_cleanup.pm

```diff
@@ -51,11 +51,11 @@ sub snapper_cleanup {
     assert_script_run("snapper list");
     clear_console;
     script_run($btrfs_fs_usage);
-    # Get actual exckusive disk space to verify exclusive disk space is taken into account
+    # Get actual exclusive disk space to verify exclusive disk space is taken into account
     my $qgroup_excl_space = get_space("btrfs qgroup show  / --raw | grep 1/0 | awk -F ' ' '{print\$3}'");
     if ($qgroup_excl_space > $exp_excl_space) {
-        my $msg = "bsc#998360: Exclusive space is above user-defined limit: "
-          . "$exp_excl_space (expected exclusive disk space) < $qgroup_excl_space (expected exclusive disk space)";
+        my $msg = "bsc#998360: qgroup 1/0: Exclusive space is above user-defined limit:\n"
```

Comment:
what does `qgroup 1/0` mean?

Reply:
This is snapper and btrfs quota magic. snapper uses quota group 1/0 for bookkeeping of disk usage by snapshots. Please note that the $qgroup_excl_space code was already present before. My diff just increases message verbosity a bit here.

Reply:
FTR: authoritative source is http://snapper.io/2016/05/18/space-aware-cleanup.html